### PR TITLE
feat(members): add reclaimHostRole functionality

### PR DESCRIPTION
**Pull request by chrisadubois**
# COMPLETES
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-485083
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-485085

## This pull request addresses
Add the ability to see canReclaimHostRole from members
update the request params

 https://confluence-eng-gpk2.cisco.com/conf/pages/viewpage.action?spaceKey=LOCUS&title=Reclaim+Host+Role

## by making the following changes

Screenshot 2023-12-19 at 12 50 19 PM
Screenshot 2023-12-19 at 12 50 14 PM

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

- performed manual tests (see screenshots)
- performed unit testing (see tests)
- used window.webex to perform tests with this as well as with native client to ensure it was all properly passed

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request
- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/test/unit/spec/member/index.js [hunk 1]
```diff
@@ -17,13 +17,7 @@ describe('member', () => {
     assert.exists(member.supportsInterpretation);
     assert.exists(member.supportsBreakouts);
     assert.exists(member.supportLiveAnnotation);
-  });
-
-  it('checks that processParticipant calls isHandRaised', () => {
```
Comment (nataliadelmar) [inline packages/@webex/plugin-meetings/test/unit/spec/member/index.js:22]: was this deleted on purpose?

**Review by nataliadelmar — APPROVED** [review]

File: packages/@webex/plugin-meetings/src/member/util.ts [hunk 2]
```diff
@@ -28,6 +28,21 @@ import ParameterError from '../common/errors/parameter';
 
 const MemberUtil: any = {};
 
+/**
+ * @param {Object} participant the locus participant
+ * @returns {Boolean}
+ * https://confluence-eng-gpk2.cisco.com/conf/pages/viewpage.action?spaceKey=LOCUS&title=Reclaim+Host+Role
```
[the remark this answers is not in the document]
Comment (chrisadubois) [reply packages/@webex/plugin-meetings/src/member/util.ts:34]: sure - although the confluence is behind SSO anyway

File: packages/@webex/plugin-meetings/src/member/index.ts [hunk 3]
```diff
@@ -11,6 +11,7 @@ import MemberUtil from './util';
  */
 export default class Member {
   associatedUser: any;
+  canReclaimHost: boolean;
```
[the remark this answers is not in the document]
Comment (chrisadubois) [reply packages/@webex/plugin-meetings/src/member/index.ts:14]: boolean field gets set in the constructor during process participant

**Comment by chrisadubois** [comment]
> This could be moved under the process participant section right?

it is

File: packages/@webex/plugin-meetings/src/member/util.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (chrisadubois) [reply packages/@webex/plugin-meetings/src/member/util.ts:34]: just as an FYI - there are numerous other places with confluence links

File: packages/@webex/plugin-meetings/test/unit/spec/member/index.js (hunk 1 above)
Comment (chrisadubois) [reply packages/@webex/plugin-meetings/test/unit/spec/member/index.js:22]: it was yes

File: packages/@webex/plugin-meetings/src/member/util.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (chrisadubois) [reply packages/@webex/plugin-meetings/src/member/util.ts]: done

**Review by sreenara — APPROVED** [review]

**Pull request merged**